report-upcoming-release-version/action.yml

```
@@ -0,0 +1,59 @@
+name: "Report release version to Slack"
+description: "Report upcoming release version to Slack"
+inputs:
+  project:
+    description: "The name of the project."
+    required: true
+  version:
+    description: "The upcoming version to release."
+    required: true
+  slack-channel:
+    description: "The name of the Slack channel."
+    default: release
+  AZURE_KV_CI_SERVICE_PRINCIPAL:
+    description: "The service principal used to authenticate to Azure."
+    required: true
+outputs:
+  channel_id:
+    description: "Channel ID where the Slack message was posted"
+    value: ${{ steps.slack-message.outputs.channel_id }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Login to Azure - Prod Subscription
```

❓ Can we always assume the type of subscription here?
`name: Login to Azure - Prod Subscription`

Maybe something like this wouldn't assume details that may not be correct.
```suggestion
    - name: Login to Azure
```